openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -5291,13 +5291,7 @@ OS:SubSurface,
        \minimum 0
        \maximum 1
        \default autocalculate
-  A7, \field Shading Control Name
```

Comment:
@jmarrec this seems like it would require some api changes.  My thought for right now is to preserve the current api and idd, just handle this at translation time.  So I think the changes to ProposedE+.idd are good, but we should not make changes to the OS IDD at this time.

Reply:
I agree. The commit with changes to OS Idd was marked as  "just for reference" for this reason